module-1/lab-api-scavenger-game/your-code/challenge_3.ipynb

```diff
@@ -26,7 +26,8 @@
     "import json\n",
```

Comment:
Always the second approach!! Great code the hardcore path. Congrats!!
Remember to keep your for loops under control!!! jejejej